frontend/test/metabase/components/TokenField.unit.spec.js

```diff
@@ -62,6 +62,21 @@ class TokenFieldWithStateAndDefaults extends React.Component {
 }
 
 describe("TokenField", () => {
+  beforeAll(() => {
+    // temporarily until JSDOM updates
```

Comment:
To try to prevent this block of code from living forever in the codebase, two suggestions:

1. making a ticket about removing the shim 
2. adding what version JSDOM should be updated to, which would allow us to remove the shim